resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,44 @@
+define('wikia.flowTracking', ['wikia.tracker', 'mw'], function (tracker, mw) {
+
```

Comment:
Do you think it's proper place to add constants while we are passing flows as arguments here?